ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/netty/PeerConnectionRegistry.java

```diff
@@ -66,7 +67,11 @@ public int size() {
   }
 
   public boolean isAlreadyConnected(final BytesValue nodeId) {
-    return connections.containsKey(nodeId);
+    return getConnectionForPeer(nodeId).isPresent();
```

Comment:
why did this need to change?

Reply:
Kinda preferred re-using the same interface as was now exposed ... but have changed it back.